src/doc/reference/glossary/parallelism.md

```diff
@@ -2,12 +2,7 @@
 title: Parallelism
 tags: parallelism
 ---
-
-The ratio $T_1/T_{\infty}$ of the work to the span gives the *parallelism* of a parallel
-computation. We can view the parallelism from three perspectives. As a ratio, the
-parallelism denotes the average amount of work that can be performed in parallel
-for each step along the critical path. As an upper bound, the parallelism gives the
-maximum possible speedup that can be achieved on any number of processors. Perhaps 
-most important, the parallelism provides a limit on the possibility of attaining
-perfect linear speedup. Specifically, once the number of processors exceeds the
-parallelism, the computation cannot possibly achieve perfect linear speedup.
\ No newline at end of file
+The ratio of {% defn "work" %} to {% defn "span" %}, which is the
+largest speedup an application could
+possibly attain when run on an infinite
+number of {% defn "processor", "processors" %}.
```

Comment:
I don't know whether it's worth clarifying that parallelism is the maximum possible parallel speedup *on an ideal parallel machine*.  On real hardware, maximum parallel speedup can be affected by other complications, like caching and memory effects.  But I'm not sure if the glossary should get into this nuance.

Reply:
OK I will hold off and err on side of simplicity.